modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi.adoc
+
+[id="csi-drivers-supported_{context}"]
+= CSI drivers supported by {product-title}
+
+{product-title} installs certain CSI drivers by default, giving users storage options that are not possible with in-tree volume plug-ins.
+
+To create CSI-provisioned persistent volumes that mount to these supported storage assets, {product-title} installs the necessary CSI driver Operator, the CSI driver, and the required storage class in the `openshift-cluster-csi-drivers` namespace by default.
+
+The following table describes the CSI drivers that are installed with {product-title} and which CSI features they support, such as volume snapshots, cloning, and resize. 
+
+.Supported CSI drivers and features in {product-title}
```

Comment:
I think we need to add GCE CSI driver. It did not support clone and ephemeral volumes currently.

Reply:
Oh, right...added, thanks!